command-line/dot-profile.md

```diff
@@ -12,7 +12,11 @@ Here are links to lessons that should be completed before this lesson:
 
 ### Motivation
 
-Learn about how to customize a computer's environment 
+With knowledge about dot profile, you can customize your computer’s environment.
+
+Apart from having a home directory to create and store files, users need an environment that gives them access to the tools and resources. When a user logs in to a system, the user’s work environment is determined by the initialization files. These initialization files are defined by the user’s startup shell, which can vary depending on the release. The default initialization files in your home directory enable you to customize your working environment.
```

Comment:
Lines 17 and 18 are good to have, but it’s a lot for this section? Can you move them to the beginning of ### Lesson? I think they would make a good intro.